bloom-housing/bloom
# fix: if programs is null no longer errors

This PR addresses #4357 


## Description
When an application's program data was null, it would cause the a crash in the api when trying to do an export. This fixes that

## How Can This Be Tested/Reviewed?
- create an application that has `null` from program data
- attempt to export the application data for the listing the application was added to
- the app should no longer error and should return an export

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
